gtk/ui/gtk3/PrefsDialog.ui

```diff
@@ -2095,7 +2095,7 @@ To add a new primary URL, add it after a blank line.</property>
       <widget name="max_total_peers_label"/>
     </widgets>
   </object>
-  <object class="GtkSizeGroup" id="privary_page_labels_width_group">
+  <object class="GtkSizeGroup" id="privacy_page_labels_width_group">
```

Comment:
This is wrong in the gtk4 version too

Reply:
what's the address of the gtk repository?

Reply:
Zero hit on Google, and I can't find it in the gtk source code:
https://www.google.com/search?rls=en&q=gtk4+%22privary_page_labels_width_group%22&sa=X&dpr=1

Reply:
That's a link I pasted

Reply:
Ah I get it. It's not in the "gtk" project, but in our code since 4 days ago with #3939. In my defence, my PR came earlier.

Reply:
I think @livings124 says, that recently introduced UI files for GTK4 port of transmission contains the same typo.
It's not inside GTK itself, no need to report it to GTK developers at GNOME's gitlab.
Just rerun your proofreading tool on `transmission/gtk/ui/gtk4`, that's it.